src/Microsoft.AspNetCore.StaticFiles/ResponseCacheLocation.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.StaticFiles
+{
+    /// <summary>
+    /// Determines the value for the "Cache-control" header in the response.
+    /// </summary>
+    public enum ResponseCacheLocation
+    {
+        /// <summary>
+        /// Cached in both proxies and client.
+        /// Sets "Cache-control" header to "public".
+        /// </summary>
+        Any = 0,
```

Comment:
It is what we have in MVC's reponse cache attribute so maybe it's good to be consistent. However, I think it may be more explicit if we just named the Enums to what is being set, i.e. Public, Private, None/No-Cache